.vscode/settings.json

```diff
@@ -20,9 +20,6 @@
   "jest.showCoverageOnLoad": true,
   "jest.runAllTestsFirst": false,
   "[typescript]": {
-    "editor.defaultFormatter": "dbaeumer.vscode-eslint"
+    "editor.defaultFormatter": "esbenp.prettier-vscode"
```

Comment:
is there some equivalent for javascript since there are still a considerable number of javascript files in our codebase

Reply:
with a plan to remove it in the future.